modules/local/sage/germline/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: sage_germline
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::hmftools-sage=3.4.3
```

Comment:
3.4.4

Reply:
and below. I won't comment on every hmftools-sage conda recipe for this

Reply:
Updated this and others in https://github.com/nf-core/oncoanalyser/pull/72